platform/security/pdp/security-pdp-xacmlrealm/src/main/java/ddf/security/pdp/xacml/processor/BalanaPdp.java

```diff
@@ -104,13 +110,20 @@ public BalanaPdp(String relativeXacmlPoliciesDirectoryPath) throws PdpException
     }
 
     /**
+     * Disable the default constructor
+     */
+    private BalanaPdp() {
```

Comment:
From [the Java tutorial](https://docs.oracle.com/javase/tutorial/java/javaOO/constructors.html)...

> You don't have to provide any constructors for your class, but you must be careful when doing this. The compiler automatically provides a no-argument, default constructor **for any class without constructors.**

*Emphasis mine.*